src/common-utils/scripts/devcontainers.zsh-theme

```diff
@@ -24,3 +24,19 @@ __zsh_prompt() {
     unset -f __zsh_prompt
 }
 __zsh_prompt
+
+# Function to set the terminal title to the current command
```

Comment:
Same comment here, can we condition this to use `TERM_PROGRAM`?

Reply:
Checking $TERM now